scripts/check_frontend_test_coverage.py

```diff
@@ -150,12 +161,43 @@ def check_not_fully_covered_filenames_list_is_sorted() -> None:
         sys.exit(1)
 
 
-def check_coverage_changes() -> None:
+def check_if_file_should_be_checked(
+    file_path: str,
+    files_to_check: Optional[List[str]] = None
+) -> bool:
+    """Check if the file should be checked for coverage changes.
+
+    Args:
+        file_path: str. The path of the file to check.
+        files_to_check: list(str)|None. The list of files to check for
+            coverage changes.
+
+    Returns:
+        bool. Whether the file should be checked for coverage changes.
+    """
+    if files_to_check is None:
+        return True
+
+    for file_to_check in files_to_check:
+        file_to_check_name = os.path.split(file_to_check)[1]
```

Comment:
```suggestion
        _, file_to_check_name = os.path.split(file_to_check)
```

I recommend writing it this way to make clear that `split(...)` only ever returns two elements

Reply:
Done.